tests/unit/cmd/iac/test_scan_diff.py

```diff
@@ -9,7 +9,7 @@
 from tests.unit.conftest import _IAC_SINGLE_VULNERABILITY, my_vcr
 
 
-@my_vcr.use_cassette("test_iac_scan_no_argument")
```

Comment:
I think you forgot to remove the old cassettes.

Reply:
I renamed the previous cassettes. `test_iac_scan_no_argument` is used in `test_scan_all` and `test_scan`.